src/sidebar/components/ShareDialog/ExportAnnotations.tsx

```diff
@@ -228,7 +233,14 @@ function ExportAnnotations({
                       key={exportFormat.value}
                       value={exportFormat}
                     >
-                      {exportFormat.name}
+                      <div className="flex-col gap-y-2">
+                        <div className="font-bold" data-testid="format-name">
+                          {exportFormat.name}
+                        </div>
+                        <div data-testid="format-description">
+                          {exportFormat.description}
+                        </div>
```

Comment:
We probably want to limit the width here and let the description wrap if needed. For the JSON description for example, I think that's OK if it wraps onto two lines.

Reply:
I'm struggling to make it wrap many lines when the container's width is set to a smaller value.

I need to debug it a bit further, but since it doesn't look terrible even with a width set to the longest line, I'm going to go ahead and merge it, and continue debugging separately.